src/server/blockchain/config.js

```diff
@@ -0,0 +1,7 @@
+require('dotenv').config();
```

Comment:
The environment variables were already set in the main file here: https://github.com/lazaruslabs/culturestake/blob/master/src/server/index.js#L13 - there is not need to call the function twice.

Reply:
I think I was echoing what's in `database/config.js`, but can remove

Reply:
@ana0 I see! In database we have to do it like that as the file is imported by the `sequelize-cli` command line application which does not support `import` syntax.